src/java/org/apache/cassandra/security/FileBasedSslContextFactory.java

```diff
@@ -174,6 +168,28 @@ protected TrustManagerFactory buildTrustManagerFactory() throws SSLException
         }
     }
 
+    private KeyManagerFactory getKeyManagerFactory(final String keystore, final String keystorePassword) throws SSLException
+    {
+        try (InputStream ksf = Files.newInputStream(Paths.get(keystore)))
+        {
+            final String algorithm = this.algorithm == null ? KeyManagerFactory.getDefaultAlgorithm() : this.algorithm;
```

Comment:
Maybe a debug log statement mentioning that we are falling back to a default algorithm?